src/components/EasySlider/EasySliderContainer.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import EasySlider from "./EasySlider";
+import { settings } from "./settings/sliderSettings";
+import { mainApi } from "../../api/api.js";
+
+const EasySlideContainer = () => {
+  const [bakeries, setBaker] = useState([]);
+
+  useEffect(() => {
+    (async function () {
+      try {
+        let bakeries = await mainApi.getBakeries();
```

Comment:
which type `getBakeries` has to return so async/await works fine?

Reply:
We work here with promises and js. So if we're talking about real types from Object-oriented programming, I had to use kind of a ts to control it. We also have propTypes to React, but this is also the one thing that I forgot to use in this exercise.   So, if we need restrictions actually we need to get type an Object.